modules/experimental/pages/cc-blast.adoc

```diff
@@ -0,0 +1,15 @@
+= CC Blast
+
+`Detachment ability`
+
+Some forces are able to disrupt their close combat opponents, for example by infecting them with Nurgle's Rot.
+
+---
+
+Where your detachment has the {cc-blast} ability and takes part in a close combat:
+
+* After rolling to resolve the close combat, every enemy detachment that was in base contact with your detachment gains one Blast marker. This effect does not stack with other detachments with the {cc-blast} ability.
```

Comment:
```suggestion
* After you roll for assault resolution in this close combat, place one additional Blast marker on each enemy detachment that had any units in base contact with units in this detachment.
This effect does *not* stack with other detachments that also have the {cc-blast} ability.
```

Reply:
Yes and no!

'Additional' infers that there are already Blast markers being placed on the enemy detachments, but that happens in the _next_ step and only if you win the close combat. I think this can be easily fixed by removing 'additional'. 

So:

```suggestion
* After you roll for assault resolution in this close combat, place one Blast marker on each enemy detachment that had any units in base contact with units in this detachment.
This effect does *not* stack with other detachments that also have the {cc-blast} ability.
```

Although... maybe we should just refer to the step directly? In which case:

```suggestion
* At the end of xref:roll-to-resolve-the-close-combat.adoc[], place one Blast marker on each enemy detachment that had any units in base contact with units in this detachment.
This effect does *not* stack with other detachments that also have the {cc-blast} ability.
```

'After rolling to resolve the close combat' was intentionally echoing the step title 'CC3. Roll to resolve the close combat', but if you think 'After you roll for assault resolution in this close combat' works better, I have no objection.

Reply:
> 'Additional' infers that there are already Blast markers being placed on the enemy detachments, but that happens in the next step and only if you win the close combat. I think this can be easily fixed by removing 'additional'.

Ah yes, I see your point.

> 'After rolling to resolve the close combat' was intentionally echoing the step title 'CC3. Roll to resolve the close combat', but if you think 'After you roll for assault resolution in this close combat' works better, I have no objection.

The way that we refer to assault resolution varies a bit — we haven't found a single way to do it that is unambiguous and robust in all cases. While the topic titles are 'CC3. Roll to resolve the close combat' and 'FF3. Roll to resolve the firefight', the titles of the procedures within are 'Assault resolution roll (close combat)' and 'Assault resolution roll (firefight)', respectively.

IIRC, we sometimes need to refer to the 'assault resolution roll' (or the 'roll for assault resolution') to cover both close combat and firefight. (I think the main instances were on the Fate cards that I've started on, but are probably elsewhere in the rules too — I can't remember right now.)

We seem to have several options:

1. 'After you roll to resolve the close combat'
2. 'After you roll for assault resolution in this close combat' (or perhaps 'After you make the assault resolution roll in this close combat')
3. 'At the end of the xref:main-rules:roll-to-resolve-the-close-combat.adoc[] step'

I suspect that players will 'get it' in any of these cases, but: I've just noticed that we'll also have to be careful not to leave room for players to think that this ability means that when there is a draw, this ability will add a Blast marker *each time* on top of the one that they place as a result of the draw (which happens 'after the roll').

For this reason especially, I think that your suggestion for (3) seems best over all — in part because the 'at the end of the step' phrasing leaves less room for interpretation than the 'after the roll' phrasing. So:

```suggestion
* At the end of the xref:main-rules:roll-to-resolve-the-close-combat.adoc[] step, place one Blast marker on each enemy detachment that had any units in base contact with units in this detachment.
This effect does *not* stack with other detachments that also have the {cc-blast} ability.
```